packages/block-library/src/list/test/edit.native.js

```diff
@@ -347,7 +347,10 @@ describe( 'List block', () => {
 		// backward delete
 		const listItemField =
 			within( listItemBlock ).getByLabelText( /Text input. .*Two.*/ );
-		changeAndSelectTextOfRichText( listItemField, 'Two' );
+		changeAndSelectTextOfRichText( listItemField, 'Two', {
+			initialSelectionStart: 0,
+			initialSelectionEnd: 3,
+		} );
```

Comment:
> It took me a bit to understand that this call is just for changing the selection to the beginning of the text 😅. I'm thinking that would be helpful to have another helper just for selecting the text, at least it would help to make this part easier to understand, WYT?

Referenced in the inline comment above this line, the overarching goal was to merely place the cursor at the end of the current input value. 

The current logic is selecting the entirety of the text to avoid the text becoming `TwoTwo` when passing a duplicative "Two" as the "new" value. I realize now that if we apply of [your other suggestion](https://github.com/WordPress/gutenberg/pull/49732#discussion_r1164397481), this could become `changeAndSelectTextOfRichText( listItemField, '' )`; i.e. appending an empty string to the current value which will result in the cursor at the end of the current value.

Reply:
I was mistaken. I reversed the requirement of the test in my head. It requires the cursor at the beginning of the input, so it will still require passing an option: 

```js
changeAndSelectTextOfRichText( listItemField, '', {
  selectionStart: 0,
} );
```